### PR TITLE
Feature: VSI from existing boot volume

### PR DESCRIPTION
**Pull request by deepaksibm**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Comment by deepaksibm** [comment]
```
tfpt TestAccIBMISVolumeDatasource_basic
=== RUN   TestAccIBMISVolumeDatasource_basic
--- PASS: TestAccIBMISVolumeDatasource_basic (77.15s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     78.774s


tfpt TestAccIBMISVolumeDatasource_from_snapshot
=== RUN   TestAccIBMISVolumeDatasource_from_snapshot
no message sent
no message sent
no message sent
no message sent
no message sent
--- PASS: TestAccIBMISVolumeDatasource_from_snapshot (336.55s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     337.511s

tfpt TestAccIBMISVolume_snapshot
=== RUN   TestAccIBMISVolume_snapshot
no message sent
no message sent
no message sent
--- PASS: TestAccIBMISVolume_snapshot (310.01s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     311.541s


tfpt TestAccIBMISVolume_basic
=== RUN   TestAccIBMISVolume_basic
--- PASS: TestAccIBMISVolume_basic (116.43s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     117.381s


$tfpt TestAccIBMISInstance_ByVolume
=== RUN   TestAccIBMISInstance_ByVolume
no message sent
no message sent
no message sent
no message sent
no message sent
no message sent
no message sent
no message sent
--- PASS: TestAccIBMISInstance_ByVolume (352.09s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     353.056s


tfpt TestAccIBMIsVolumesFromSnapshotDataSourceBasic
=== RUN   TestAccIBMIsVolumesFromSnapshotDataSourceBasic
no message sent
no message sent
--- PASS: TestAccIBMIsVolumesFromSnapshotDataSourceBasic (311.19s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     312.158s
```

File: ibm/service/vpc/data_source_ibm_is_volume.go [hunk 1]
```diff
@@ -109,38 +206,6 @@ func DataSourceIBMISVolume() *schema.Resource {
 							Computed:    true,
 							Description: "Link to documentation about this status reason",
 						},
-
-						isVolumeHealthReasons: {
```
[the remark this answers is not in the document]
Comment (deepaksibm) [reply ibm/service/vpc/data_source_ibm_is_volume.go:113]: This value was not being set in the code. It is not mentioned in docs as well.  do we still need it ?

[the remark this answers is not in the document]
Comment (deepaksibm) [reply ibm/service/vpc/data_source_ibm_is_volume.go:113]: added back with a deprecation note.

[the remark this answers is not in the document]
Comment (hkantare) [reply ibm/service/vpc/data_source_ibm_is_volume.go:113]: Then we can delete it

[the remark this answers is not in the document]
Comment (deepaksibm) [reply ibm/service/vpc/data_source_ibm_is_volume.go:113]: deleted

**Pull request merged**